spec/commands/zrange_spec.rb

```diff
@@ -46,6 +46,10 @@
     @redises.zrange(@key, 5, -1).should == []
   end
 
+  it 'returns entire list when start is too small' do
+    @redises.zrange(@key, -5, -1).should == %w[Washington Adams Jefferson Madison]
+  end
```

Comment:
@trotzig good call, just added these cases and led to much simpler implementation 👍